examples/finnegans_wake_demo/run_federated_ursula.py

```diff
@@ -32,16 +32,16 @@
 # TODO: Use real path tooling here.
 SHARED_CRUFTSPACE = "{}/examples-runtime-cruft".format(os.path.dirname(os.path.abspath(__file__)))
 CRUFTSPACE = "{}/{}".format(SHARED_CRUFTSPACE, MY_REST_PORT)
-DB_NAME = "{}/database".format(CRUFTSPACE)
+db_filepath = "{}/database".format(CRUFTSPACE)
```

Comment:
I was wondering why the demo hung here.

Reply:
Hmm, I dont understand what you are indicating here. :question:

Reply:
Nor I.  

We probably actually need some rudimentary tests for the demo (did we have an Issue this once upon a time?  I can't find it now.).